Sources/StreamChatUI/Generated/L10n.swift

```diff
@@ -166,8 +166,8 @@ internal enum L10n {
 
 extension L10n {
   private static func tr(_ table: String, _ key: String, _ args: CVarArg...) -> String {
-    let format = BundleToken.bundle.localizedString(forKey: key, value: nil, table: table)
-    return String(format: format, locale: Locale.current, arguments: args)
+     let format = Appearance.default.localizationProvider(key, table)
```

Comment:
We should add a TODO here, this using `Appearance.default` will prohibit using Appearance injection